ios/RCTConvert+BitmovinPlayer.swift

```diff
@@ -1183,3 +1203,8 @@ internal struct RNBufferLevels {
     let audio: BufferLevel
     let video: BufferLevel
 }
+
+internal enum RNPictureInPictureAction {
+    case togglePlayback
+    case seek
+}
```

Comment:
You can give raw values to the enum to enable simpler String<>RNPictureInPictureAction conversion.

```suggestion
internal enum RNPictureInPictureAction: String {
    case togglePlayback = "TogglePlayback"
    case seek = "Seek"
}
```

Reply:
Ah cool, thanks!